src/vote.rs

```diff
@@ -245,6 +265,19 @@ impl<T: Proposition> Debug for SignedVote<T> {
 }
 
 impl<T: Proposition> SignedVote<T> {
+    pub fn candidate(&self) -> Candidate<T> {
+        match &self.vote.ballot {
+            Ballot::SuperMajority { votes, .. } => VoteCount::count(votes, &self.vote.faulty_ids())
+                .candidate_with_most_votes()
+                .map(|(candidate, _)| candidate.clone())
+                .unwrap_or_default(),
+            _ => Candidate {
+                proposals: self.proposals(),
+                faulty: self.vote.faulty_ids(),
+            },
+        }
```

Comment:
When merging / proposing, we want to take all proposals, but when we sent super-majority, the proposals we want are from the candidate with most votes.